src/modules/launcher/PowerLauncher/App.xaml.cs

```diff
@@ -103,6 +103,8 @@ private void OnStartup(object sender, StartupEventArgs e)
                 _mainVM = new MainViewModel(_settings);
                 _mainWindow = new MainWindow(_settings, _mainVM);
                 API = new PublicAPIInstance(_settingsVM, _mainVM, _themeManager);
+                _settingsWatcher = new SettingsWatcher(_settings, _themeManager);
```

Comment:
Previously the watcher was initialized after `PluginManager.InitializePlugins(API);`, wouldn't be better to keep it that way or is there a specific reason to move it here?

Reply:
The watcher updates plugins metadata with data from `settings.json`. If we do not put it above `PluginManager.InitializePlugins(API);`, plugins will be initialized with default settings. And it would be possible that disabled plugins are loaded.

Reply:
Renamed `SettingsWatcher` to `SettingsReader` as `SettingsWatcher` does not imply that the first read will be made.